pkg/authorization/bootstrap/policy.go

```diff
@@ -54,6 +56,13 @@ func clusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule(bootstrappolicy.Read...).Groups(tenancy.GroupName).Resources("clusterworkspacetypes").RuleOrDie(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: SystemKcpWorkspaceBootstrapper},
```

Comment:
@sttts @s-urbaniak PTAL

Reply:
let's please clearly document why a separate role is needed vs "just using cluster-admin".

Reply:
(as a godoc)

Reply:
as far as i understand it is because we want a dedicated group vs cluster-admin because .... (?)